### PR TITLE
proceed even if non-ASCII filenames are encountered

### PR DESCRIPTION
**Pull request by olafhering**
Signed-off-by: Olaf Hering <olaf@aepfle.de>

### CODE REVIEW

File: DuplicatesCheck.py [hunk 1]
```diff
@@ -58,7 +58,12 @@ def check(self, pkg):
 
             partition = get_prefix(one)
 
-            st = os.stat(pkg.dirName() + '/' + one)
+            try:
+                st = os.stat(pkg.dirName() + '/' + one)
+            except UnicodeError as e:  # e.g. non-ASCII, C locale, python 3
+                Filter.printError(pkg, 'inaccessible-filename', one, e)
```
Comment (dirkmueller) [inline DuplicatesCheck.py]: this is an unrelated warning to the main functionality of DuplicatesCheck, and its duplicating an existing check for utf8-compatible filenames. was the error that occurred here a file that was properly utf8 encoded (so is it a locale issue) or was it invalid in utf8?

Comment (olafhering) [reply DuplicatesCheck.py]: The filename was not utf8 but some other encoding.

Comment (dirkmueller) [reply DuplicatesCheck.py]: ok, so its a duplicate then. canwe just rely on the other check then? I don't think i tmakes sense to raise two different errors on the same issue.

**Comment by olafhering** [comment]
I rest my case. It was solved by just not creating non-utf8 path names.
